.github/scripts/build-ios.sh

```diff
@@ -28,4 +27,8 @@ npm start &
 log "command" "npx pod-install" # To prevent "AsyncStorage is null." https://react-native-async-storage.github.io/async-storage/docs/install/
 npx pod-install
 log "command" "npm run ios"
-npm run ios
+if [ $BUILD_TOOL == 'expo' ]; then
+  npx expo start --ios -p 19001
```

Comment:
We don't really need this line since we are not adding the test for iOS, but with this line, but with this line, we can reproduce the Expo issue https://github.com/expo/expo-cli/issues/4716